exporters/otlp-http/trace/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id("otel.java-conventions")
+    // TODO: uncomment once ready to publish
+    // id("otel.publish-conventions")
+
+    id("otel.animalsniffer-conventions")
+    id("com.github.johnrengelman.shadow")
+}
+
+description = "OpenTelemetry Protocol HTTP Trace Exporter"
+otelJava.moduleName.set("io.opentelemetry.exporter.otlp.http.trace")
+
+dependencies {
+  api(project(":sdk:trace"))
+
+  implementation(project(":exporters:otlp:common"))
+
+  implementation("com.squareup.okhttp3:okhttp")
+  implementation("com.squareup.okhttp3:okhttp-tls")
+  implementation("com.squareup.okio:okio")
+
+  testImplementation(project(":sdk:testing"))
+
+  testImplementation("com.linecorp.armeria:armeria-junit5")
+}
+
+tasks {
+  shadowJar {
+    minimize()
+
+    relocate("okhttp3", "io.opentelemetry.internal.shaded.okhttp3")
+    relocate("okio", "io.opentelemetry.internal.shaded.okio")
```

Comment:
We also have to relocate kotlin as well unfortunately.

Reply:
Not applicable now that I incorporated the feedback not to publish a shadow jar.